nursery/create-bits-job.yml

```
@@ -0,0 +1,21 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: create BITS job
+    namespace: communication
```

```suggestion
    namespace: communication/http/client
```

> BITS supports transfers over SMB, HTTP and HTTPS. 
https://en.wikipedia.org/wiki/Background_Intelligent_Transfer_Service

but i suspect we'd only really see this for HTTP(S)?